include/z64pause.h

```diff
@@ -98,9 +124,9 @@ typedef struct {
     /* 0x01D6 */ u16 debugState;
     /* 0x01D8 */ Vec3f eye;
     /* 0x01E4 */ u16 mainState;
-    /* 0x01E6 */ u16 mode;
+    /* 0x01E6 */ u16 nextPageMode; // (2 * prev pageIndex) + (scroll left ? 1 : 0)
     /* 0x01E8 */ u16 pageIndex; // "kscp_pos"
```

Comment:
coming form the perspective of someone who doesnt understand this mechanism all too well, the random string of code next to the member variable isnt really telling me anything. id either use words to describe it or just move that explanation somewhere else maybe?